src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -26,11 +26,27 @@
     private BlockNetworkUtil() {
     }
 
+    /**
+    * getResultSide
+    * @param block - the block
```

Comment:
`@param block - the block`
Again, what is this supposed to mean? You're not explaining anything, you're just rewriting it. As michael mentioned, your javadoc is supposed to EXPLAIN the method, not rephrase it. Somethig like
`@param block The block that is to be rotated`